test/ng/directive/formSpec.js

```diff
@@ -523,6 +523,22 @@ describe('form', function() {
       expect(parent.$submitted).toBeTruthy();
     });
 
+    it('should set $submitted equal to true on child forms when parent is submitted', function() {
+      doc = jqLite(
+          '<ng:form name="parent">' +
```

Comment:
Can you please use `ng-form` et. al.? I think these tests are very old, that's why they use `ng:form`.